### PR TITLE
Implement automatic reload on source activation.

### PR DESCRIPTION
**Pull request by NexAdn**
As requested by issue #39.

Signed-off-by: NexAdn <nexadn@yandex.com>

### CODE REVIEW

**Comment by S-Marais** [comment]
Perfect. Does exactly what I want. Thx!

File: src/plugin/main.c [hunk 1]
```diff
@@ -200,6 +202,13 @@ static bool reload_button_clicked(obs_properties_t *props, obs_property_t *prope
 	return true;
 }
 
+static void reload_on_scene(void* vptr)
+{
+	struct browser_data* data = vptr;
+	if (data->reload_on_scene)
+		reload_button_clicked(NULL, NULL, vptr);
```
Comment (bazukas) [inline src/plugin/main.c]: Better use `browser_manager_reload_page` here

**Comment by bazukas** [comment]
Hello, great job! I have only one minor nitpick I posted in the comments, would be great if you could fix it

File: src/plugin/main.c (hunk 1 above)
Comment (bazukas) [reply src/plugin/main.c]: `reload_button_clicked` is a callback specific to button click and might do stuff other than just reload a page.

Comment (NexAdn) [reply src/plugin/main.c]: Will be done.

**Pull request merged**